### ISSUE TITLE
Rain washes windscreen banner away without ExtraFX enabled.

### ISSUE THREAD

**Issue by Vannipo** [ac-custom-shaders-patch/acc-extension-config#222]
If ExtraFX is not enabled, raindrops makes the windscreen banner transparent.
![20220302004729_1](https://user-images.githubusercontent.com/76270473/156268572-bc0c769a-a16a-47c5-a25f-79afa16f3a55.jpg)

https://user-images.githubusercontent.com/76270473/156269085-31c4784a-5815-4e2a-9d1c-d4c4fd59a9d2.mp4

**Comment by Marocco2** [comment]
Might not be a bug: Banner should be set inside DELAYED_RENDER

**Comment by Vannipo** [comment]
Yeah, might not be a bug, but since u cannot use ExtraFX in VR and many users turn ExtraFX off cause of fps, another solution for this would be nice.

**Comment by Vannipo** [comment]
And how an ExtraFX flag should help, while ExtraFX is turned off?
